SPECS/libxcrypt/libxcrypt.spec

```diff
@@ -142,12 +142,7 @@ Provides:       %{name}-common = %{version}-%{release}
 Obsoletes:      %{name}-compat < %{version}-%{release}
 %endif
 # We need a version of glibc, that doesn't build libcrypt anymore.
-#Requires:       glibc%{?_isa}         >= %{glibc_minver}
-%if %{with override_glibc}
-# Require a specific glibc version so the post macro is compatible.
-BuildRequires:  glibc-devel = %{glibcversion}
-Requires:       glibc = %{glibcversion}
-%endif
+Requires:       glibc%{?_isa}         >= %{glibc_minver}
```

Comment:
Not that it matters much but why such a huge gap before `>=`? The rest of the spec seems to be using a single space, so it caught my eye.

Reply:
The huge gap is actually in the upstream fedora spec, and is also present in many of the other lines. https://src.fedoraproject.org/rpms/libxcrypt/blob/f31/f/libxcrypt.spec

I suspect some linting was applied to our version of the spec, so the spaces and newlines were condensed/replaced.

I will adjust this line to match the rest of the spec.